src/components/Firestore/Collection.tsx

```diff
@@ -70,14 +72,17 @@ export function withCollectionState(
       // This works around an issue in React legacy mode that causes filters to
       // be ignored and listeners leaking.
       // TODO(yuchenshi): remove after switching to blocking / concurrent mode.
-      { startWithValue: { docs: [] } as any }
+      {
+        suspense: true,
+        initialData: { docs: [] } as any,
```

Comment:
Please drop this and the comment block above. See the comments above for why.

Reply:
Done.